### PR TITLE
[NAVAND-1714] Upgrade services-cli

### PR DESCRIPTION
**Pull request by holuss**
JIRA: [NAVAND-1714](https://mapbox.atlassian.net/browse/NAVAND-1714)

What has been done:
* handle string input: -j "example json file"
* exit with code 1 when failed verification
* exit also with code 1 on back conversion: -c
* pretty print result: -p
* updated tests

Example:
Screenshot 2024-06-06 at 13 18 19

[NAVAND-1714]: https://mapbox.atlassian.net/browse/NAVAND-1714?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by VysotskiVadim — COMMENTED** [review]
LGTM 👍 
Let's also wait for review from Directions API, as they're going to adopt this validator, and merge in case of positive feedback

**Review by tomaszrybakiewicz — APPROVED** [review]
LGTM. Like @VysotskiVadim said, let's wait for the Directions API team to review this before merging.
@holuss You might want to add the API engineers as reviewers and/or ping them on slack.

**Comment by holuss** [comment]
@VysotskiVadim I've decided to merge because I will be for a long time on TPO. If there will be a need to update, please open new PR.

**Pull request merged**
